src/app/vault/ciphers.component.ts

```diff
@@ -80,6 +86,90 @@ export class CiphersComponent extends BaseCiphersComponent implements OnDestroy
         this.didScroll = this.pagedCiphers.length > this.pageSize;
     }
 
+    async loadOranizations(): Promise<Map<string, string>> {
+        const organizations = await this.userService.getAllOrganizations();
+        const organizationNames = new Map<string, string>();
+
+        if (organizations.length > 0) {
+            organizations.forEach(organization => {
+                organizationNames.set(organization.id, organization.name);
+            });
+        }
+
+        return organizationNames;
+    }
+
+    getOrganizationById(c: CipherView): string {
+        if (c.organizationId != null) {
+            return this.organizationNames.get(c.organizationId);
+        }
+        return 'Personal';
```

Comment:
This should be translatable, maybe use 'myVault'
```suggestion
        return this.i18nService.t('myVault');
```

Reply:
I think 'My Vault' works fine since there is already a translation for this. 👍🏻